sdks/python/apache_beam/io/fileio.py

```diff
@@ -1037,8 +1037,9 @@ def __init__(self,
                num_shards=0,
                shard_name_template=None,
                coder=coders.ToStringCoder(),
-               compression_type=CompressionTypes.AUTO):
-    """Initialize a TextFileSink.
+               compression_type=CompressionTypes.AUTO,
+               header=None):
+    r"""Initialize a TextFileSink.
```

Comment:
Note the '\n' character in the description.  If there is no r in the beginning it would give style errors.